cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AdminParameters.java

```diff
@@ -32,13 +32,12 @@
   private Integer _concurrentInterBrokerPartitionMovements;
   private Integer _concurrentLeaderMovements;
   private Integer _reviewId;
-  private final boolean _twoStepVerificationEnabled;
+  private boolean _twoStepVerificationEnabled;
```

Comment:
(Not introduce by this patch)this field is one-time use(in `initParameters()`), looks like we do not need to store it in a field, just like what `RebalanceParameters` does. also apply to `PauseResumeParameters`/`StopProposalParameters`.